### PR TITLE
category_encoders 2.6.1

### PR DESCRIPTION
**Pull request by Arishamays1**

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,35 +1,35 @@
...
-    - {{ PYTHON }} -m pip install --no-deps -vvv .
+  skip: true  # [py<38]
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
 
 requirements:
   host:
     - python
     - pip
     - wheel
+    - numpy >=1.14.0
```
Comment (cbouss) [inline recipe/meta.yaml]:
```suggestion
```
numpy is not needed in host when only the python interface is used.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,35 +1,35 @@
...
   run:
     - python
     - numpy >=1.14.0
     - scikit-learn >=0.20.0
     - scipy >=1.0.0
-    - statsmodels >=0.9.0
     - pandas >=1.0.5
     - patsy >=0.5.1
+    - statsmodels >=0.9.0
+    - importlib_resources  # [py<39]
```
Comment (cbouss) [inline recipe/meta.yaml]:
```suggestion
```
I don't see importlib_resrouces here: https://github.com/scikit-learn-contrib/category_encoders/blob/2.6.1/setup.py#L34

File: recipe/meta.yaml [hunk 3]
```diff
@@ -41,7 +41,7 @@ test:
     - category_encoders
   commands:
     - python -m pip check
-    - pytest -vv -k "not (pandas_index or truncated_index or test_extraValue)"
+    - python -W ignore -m pytest src -vv --cov=category_encoders --cov-branch -k="not (pandas_index or truncated_index)" --cov-report=term-missing:skip-covered --no-cov-on-fail --cov-fail-under=8
```
Comment (cbouss) [inline recipe/meta.yaml]: All the cov options can be omitted. We usually don't generate coverage reports.

File: recipe/meta.yaml (hunk 2 above)
Comment (Arishamays1) [reply recipe/meta.yaml]: Thanks! I think I was looking at the master branch rather than 2.6.1!

File: recipe/meta.yaml (hunk 1 above)
Comment (Arishamays1) [reply recipe/meta.yaml]: Removed! thanks

**Comment by skupr-anaconda** [comment]
Three tests
```
FAILED tests/test_target_encoder.py::TestTargetEncoder::test_hierarchy_columnwise_compass
FAILED tests/test_target_encoder.py::TestTargetEncoder::test_hierarchy_columnwise_missing_level
FAILED tests/test_target_encoder.py::TestTargetEncoder::test_hierarchy_columnwise_postcodes
```
 have the same error:

```
>           raise TypeError(f'{package!r} is not a package')
E           TypeError: 'category_encoders.datasets._base' is not a package
```
And it's really not a package, it's a module in this folder https://github.com/scikit-learn-contrib/category_encoders/tree/master/category_encoders/datasets.
Should we skip `TestTargetEncoder`?

**Comment by sumit0190** [comment]
I verified that `large_samples_binary` fails only on `numpy 1.24.3` (used for 3.8) and `numpy 1.25.0` (used for the other Python versions); with `numpy 1.23.5` it passes. So it's a regression (of sorts) in `numpy` that should probably be reported to upstream in case it hasn't already.
@Arishamays1 can you add this clarification in the `test` section so that we can come back to it in the future, if necessary?

File: recipe/meta.yaml [hunk 4]
```diff
@@ -1,48 +1,53 @@
...
+    - numpy >=1.20.0
+    - importlib_resources # [py<39]
 
 test:
   source_files:
     - tests
   requires:
     - pip
     - pytest
+    - pytest-cov
```
Comment (sumit0190) [inline recipe/meta.yaml]: You can remove this, as it's not being used anywhere.

**Review by sumit0190 — APPROVED** [review]
LGTM, just add the clarification regarding the skipped test.

**Review by skupr-anaconda — APPROVED** [review]
LGTM but address Sumit's comments

File: recipe/meta.yaml (hunk 4 above)
Comment (Arishamays1) [reply recipe/meta.yaml]: Great removed! And added clarifying comments for the skipped test.

**Pull request merged**